docs/limitations.rst

```diff
@@ -3,26 +3,35 @@ Limitations and Surprises
 *************************
 
 Templating
-==========
+##########
 
 Before templates are passed to Jinja or Cheetah there is a pre-processing of templates happening. During pre-processing
 Cobbler replaces variables like ``@@my_key@@`` in the template. Those keys are currently limited by the regex of ``\S``,
 which translates to ``[^ \t\n\r\f\v]``.
 
 Restarting the daemon
-=====================
+#####################
 
 Once you have a Cobbler distro imported or manually added you have to make sure the source for the Kernel & initrd is
 available all the time. Thus I highly recommend you to add the ISOs to your ``/etc/fstab`` to make them persistent
 across reboots. If you forget to remount them the Cobbler daemon won't start!
 
 Kernel options
-==============
+##############
 
 The user (so you) is responsible for generating the correct quoting of the Kernel Command Line. We manipulate the
 arguments you give us in a way that we add wrapping double quotes around them when the value contains a space.
 
 The Linux Kernel describes its quoting at:
 `The kernel’s command-line parameters <https://www.kernel.org/doc/html/v5.15/admin-guide/kernel-parameters.html#the-kernel-s-command-line-parameters>`_
 
-Please consult the documentation of your operating system for how it deals with this if it is not Linux.
+Consult the documentation of your operating system for how it deals with this if it is not Linux.
+
+Special Case: Uyuni/SUSE Manager
+################################
+
+.. note:: SUSE Manager is a flavor of Uyuni. The term Uyuni refers to both pieces of software in this context.
+
+If Uyuni temporarily doesn't require Cobbler anymore, the content visible in Cobbler is deleted. Cobbler is thus not
+seen as a source of truth from the perspective of Uyuni. During the runtime of the application you may see distros,
+profiles, and systems popping on and off during the runtime of the server.
```

Comment:
I don't really get what you mean with this, don't really get the point.

In which cases "Uyuni temporarily doesn't require Cobbler anymore"?

Maybe I am missing context here.. Thanks!

Reply:
I also don't get it, I think it would be good to clarify it a bit.

Reply:
Okay this is slightly incorrect. It is only the systems are popping on and off. The reason for this is the taskomatic task `kickstart_cleanup`.

Reply:
I will correct the note.

Reply:
I still do not get that this means:

```
If Uyuni temporarily doesn't require Cobbler anymore, the content visible in Cobbler is deleted. Cobbler is thus not
seen as a source of truth from the perspective of Uyuni.
```

Could you maybe rephrase those two sentences? Particularly clarifying how "Uyuni can temporarily doesn't require Cobbler anymore"

Reply:
Well if we don't do autoinstallations Cobbler is not used and thus not needed. Since Uyuni does not see this is a source of truth, it removes the content. I don't know how to phrase it better. Suggestions welcome!

Reply:
Ok, now you mentioned Autoinstallations, I think got your point! Thanks :)

I would maybe suggest something like this:

_Uyuni uses cobbler for driving autoinstallations. When using cobbler in the context of Uyuni, you need to know that cobbler is not seen by Uyuni as the source of truth. This means, in case you don't have any autoinstallation configured in Uyuni, the content visible in cobbler is deleted.
Because of the same reason, during the runtime of cobbler you may see systems popping on and off as the content of cobbler is managed by Uyuni (in particular, the taskomatic task `kickstart_cleanup` executes cleanup on the cobbler content)_

I think this brings a little bit more context to understand the point you want to indicate. What do you think?

Reply:
Will apply this. If you think this is better then I believe it is. I am to deep in this. :d